docs/api.yml

```diff
@@ -18,26 +18,59 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/metadata"
+                $ref: '#/components/schemas/metadata'
               example:
                 {
-                  "documentId": "kjsak2",
-                  "firstName": "The name",
-                  "dob": "1999-01-01",
+                  'documentId': 'kjsak2',
+                  'firstName': 'The name',
+                  'dob': '1999-01-01',
                 }
   /{documentId}:
     get:
       operationId: get-metadata
       summary: Gets a document from S3 using the document id
       parameters:
-        - $ref: "#/components/parameters/documentId"
+        - $ref: '#/components/parameters/documentId'
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/metadata"
+                $ref: '#/components/schemas/metadata'
+  /documents:
+    post:
+      operationId: find-documents
+      summary: Searches elasticsearch for documents with given metadata
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              example: { 'firstName': 'Toaster', 'dob': '1999-01-01' }
+      responses:
+        200:
+          description: Documents
+          content:
+            application/json:
+              example:
+                {
+                  'documents':
+                    [
+                      {
+                        'documentId': '1',
+                        'index': 'documents',
+                        'score': 0.3,
+                        'metadata': { 'name': '123' },
```

Comment:
I'm assuming if we have "firstName" in the request, the response is gonna contain "firstName" rather than "name"?

Reply:
yes, it can be any metadata, there aren't any restrictions right now

Reply:
yep, so lets change that to firstName so the request and response match?